src/components/tool/guide/Instructions/StartInstructions.js

```diff
@@ -34,7 +35,7 @@ export function LyraStartInstructions({
             In the ssh session, run the following command to schedule the {jobType.toLowerCase()} job:
         </InstructionText>
         <CopyBox>
-            {`qsub${jobType === "Interactive" ? " -I" : ""} -l walltime=1:00:00 -l ${resources.join(":")}`}
+            {`qsub${jobName !== "" ? " -N \"" + jobName + "\"" : ""}${jobType === "Interactive" ? " -I" : ""} -l walltime=1:00:00 -l ${resources.join(":")}`}
```

Comment:
This template is getting a bit complicated. Each expression can probably be broken out into its own variable.

Reply:
Updated, using the pattern that you defined for the resources